libraries/core-react/src/TextField/Input/Input.tsx

```diff
@@ -92,63 +92,66 @@ type Props = {
   type?: string
   /** Read Only */
   readonly?: boolean
-}
-
-const Input = React.forwardRef<HTMLInputElement, Props>(function TextFieldInput(
-  props,
-  ref,
-) {
-  const {
-    multiline = false,
-    variant = 'default',
-    inputIcon,
-    disabled = false,
-    type = 'text',
-    ...other
-  } = props
-
-  const { handleFocus, handleBlur } = useTextField()
-
-  const as: ElementType = multiline ? 'textarea' : 'input'
-  const inputVariant = tokens[variant]
-  let spacings = tokens.spacings.comfortable
-
-  if (inputIcon) {
-    spacings = {
-      ...spacings,
-      input: {
-        ...spacings.input,
-        right: '32px',
-      },
+} & InputHTMLAttributes<HTMLInputElement>
+
+const TextFieldInput = React.forwardRef<HTMLInputElement, TextfieldInputProps>(
+  function TextFieldInput(props, ref) {
+    const {
```

Comment:
I think we found out that this way of doing things doesn't work properly with argsTable because it can't figure out default props

Reply:
Which way is that?  Prefixed with `TextField`?

Reply:
Destructing inside the function parentheses :)

Reply:
Below [ update august header](https://github.com/equinor/design-system/discussions/521)

Reply:
Changed it but it didn't seem to work...  but I can change it alas.